pkgs/development/python-modules/copykitten/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  fetchFromGitHub,
+  buildPythonPackage,
+  rustPlatform,
+  stdenv,
+  lib,
+  pytestCheckHook,
+  pillow,
+}:
+
+buildPythonPackage rec {
+  pname = "copykitten";
+  version = "1.2.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Klavionik";
+    repo = pname;
+    tag = "v${version}";
+    hash = "sha256-S4IPVhYk/o15LQK1AB8VpdrHwIwTZyvmI2+e27/vDLs=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    hash = "sha256-Qgg2S1nRwWs2O81huj1g4wq7v4G377T+V8/1rjhz1ZE=";
+  };
+
+  nativeBuildInputs = [
+    pillow
+  ];
```

Comment:
This probably should go within `dependencies`, can you also move it below `build-system`?